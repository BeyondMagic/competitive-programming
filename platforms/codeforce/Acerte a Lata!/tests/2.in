13 7 -68
-72 -77 -74 72
-52 53 45 34
66 -57 12 68
-53 95 71 -98
79 11 69 -10
100 55 -49 56
68 -1 -7 -49
42 76 -35 57
-93 -59 -82 95
25 -99 -20 71
68 58 -22 -73
-40 -44 62 -23
-44 43 4 -10
21 6 62 -51
-63 -8 -22 7
8 49 9 49
-1 15 -52 -56
-55 -94 -82 -23
75 -69 -54 1
4 41 63 -42
-14 26 -60 47
69 -91 38 39
-14 -69 34 56
-46 -71 -81 30
-32 61 -54 -78
34 63 -97 -2
0 0 0
