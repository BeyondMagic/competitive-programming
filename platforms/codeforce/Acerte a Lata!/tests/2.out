Partida #1:
Meninas: 0 ponto(s)
Meninos: 0 ponto(s)
As meninas venceram
